packages/config/src/options.ts

```diff
@@ -215,6 +215,12 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     validation: validate.isBoolean,
     isExperimental: true,
     requireRestartOnChange: 'server',
+  }, {
+    name: 'experimentalUseDefaultDocumentDomain',
+    defaultValue: null,
+    validation: (_: string, value: any) => validate.isArrayOfStrings(value) || value === null,
```

Comment:
We need to enhance this a bit, if value is not an array of strings and value is not null, we wont' have a meaningful error message / will throw an error from returning an invalid error response for the error logging in the launchpad/reporter.

If `value === null` we need to return these parts:
```js
  return {
    key: 'experimentalUseDefaultDocumentDomain'
    value,
    type: 'an array of strings or null'
  }
```

Reply:
fixed in dccf967ed9